test/csuite/schema_abort/main.c

```diff
@@ -1075,6 +1117,9 @@ main(int argc, char *argv[])
         while (stat(statname, &sb) != 0)
             testutil_sleep_wait(1, pid);
         sleep(timeout);
+
+        opts->running = false;
```

Comment:
I think you probably don't need this.  The child is running in its own process and will be forcibly killed.  If it is needed (that is, fails without this line), can you add a comment why?

Reply:
Removed `opts->running = false;`